pages/en/lb3/Controlling-data-access.md

```diff
@@ -253,45 +253,41 @@ Assuming the following ACL rules are defined:
 
 ## Authorization scopes
 
 {% include warning.html content="
-The current implementation of Authorization scopes is very low-level and may not suit all users. Please join the discussion in [issue #3339](https://github.com/strongloop/loopback/issues/3339) to help us build the best high-level API for this feature.
+The current implementation of authorization scopes is very low-level and may not suit all users. Please join the discussion in [issue #3339](https://github.com/strongloop/loopback/issues/3339) to help us build the best high-level API for this feature.
 " %}
 
 LoopBack's built-in authorization algorithm allows application to limit
-access to remote methods using the concept of scopes. The implementation
+access to remote methods using the concept of _authorization scopes_. The implementation
 is similar to [oAuth2 Access Token Scope](https://tools.ietf.org/html/rfc6749#section-3.3).
 
-Authorization scopes work in tandem with the current role/ACL-based
-authorization:
+Authorization scopes work together with role/ACL-based authorization as follows:
  1. The authorization algorithm checks whether the access token
-    was granted at least one of the scopes required by the accessed resource.
-    The request is denied as not authorized when this check fails.
+    was granted by at least one of the scopes required by the accessed resource.
+    If this check fails, then the request is denied as not authorized.
  2. Next, the ACL rules are checked to make sure the established principal
-    is allowed to access the protected resource. The request is denied
-    when this check fails.
- 3. Providing both scope and ACL checks passed, the request is authorized
+    is allowed to access the protected resource. If this check fails, then the request is denied.
+ 3. If both scope and ACL checks passed, the request is authorized
     and the remote method is executed.
 
-
-There are two steps needed to start using this feature:
+To use this feature, follow these steps:
 
  1. Define scopes required by remote methods
-
  2. Add API for creating scoped AccessTokens
 
 {% include note.html content="
-LoopBack provides a single built-in scope `DEFAULT` which is used whenever a method does not define any scopes or an access token does not contain any allowed scopes. This preserves backwards compatibility for existing applications.
+LoopBack provides the built-in scope `DEFAULT` that is used whenever a method does not define any scopes or an access token does not contain any allowed scopes. This preserves backwards compatibility for existing applications.
 " %}
 
 ### Define scopes required by remote methods
 
-The scopes are defined together with other remoting metadata via the new
-setting `accessScopes`. The user can invoke the remote method as long as their access token
-is granted at least one of the scopes listed in `accessScopes` array.
+The new model property `accessScopes` defines scopes as part of remoting metadata.
```

Comment:
IIUC, it's a value set in the model JSON file, which is what I meant by a "model property".  Apparently that is confusing, but "setting" doesn't really convey any specific meaning.  

Perhaps it would be clearer to say 
> The `methods.accessScopes` property in the model JSON file defines...